### DISCUSSION TITLE
Home Assistant utilizza troppi MB

### DISCUSSION THREAD

**Discussion by I1URY** [indomus/forum#1466] [HUB: Home Assistant]
Il Raspberry 3B+ con installato solo HA in un'ora di ha utilizzato oltre 500 MB tra download e upload, mentre se gli inibisco l'accesso ad internet, il "consumo" di tutto il resto che è collegato in rete è di circa 10 MB. Dato che per mancanza di copertura fibra utilizzo internet a consumo, è un problema perché in breve tempo si mangia tutti i GB disponibili.
Il problema è recente, perché fino a qualche tempo fa non lo faceva.
Grazie a chi potrà aiutarmi

**Comment by snakuzzo** [comment]
Scrivilo sul forum / progetto github di HA fornendo le evidenze di quello che dici. Noi non possiamo farci nulla.

**Reply by I1URY** [reply]
Grazie della velocissima risposta, ma quali evidenze? e sai indicarmi il link. Grazie

**Reply by snakuzzo** [reply]
https://community.home-assistant.io/

**Reply by I1URY** [reply]
"Risolto" scritto tra virgolette perché se dovesse succedere a qualche sfortunato come me possa evitare di dannarsi inutilmente, ho scritto sulla comunity come suggerito, senza ricevere una soluzione praticabile. Per risolvere ho formattato e ricaricato HASSIO sulla SD ricominciando dall'inizio. Tutto è tornato normale.
